modules/manifold/actions.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/actions');
```

Comment:
Reverting #56, turns out we still have to use proxy files as long as the code is under `/dist` directory (non-root directory)